Documentation/ApiOverview/Workspaces/Index.rst

```diff
@@ -136,11 +136,11 @@ Frontend implementation guidelines
          This is how simple it is to use this record in your frontend plugins
          when you do queries directly (not using API functions already using
          them)::
-
-            $res = $GLOBALS['TYPO3_DB']->exec_SELECTquery(...);
-            while (($row = $GLOBALS['TYPO3_DB']->sql_fetch_assoc($res))) {
+         
+            $statement = $queryBuilder->execute();
```

Comment:
Here we should use `$result`

Reply:
ok, but it's also called $statement in the various examples of the QueryBuilder section of the documentation. ;)

Reply:
Yeah and while this is technically correct (you get an instance of a `Statement`) class, it is not really intuitive to do something with a "statement" after you have executed it. It makes way more sense to do something with a "result". And yes, the other docs could have an update here IMO.

Reply:
Ok, it's changed for this code example.